### PR TITLE
Make URL builder use site_url() to get correct http scheme

### PR DESCRIPTION
**Pull request by topher1kenobe**
@westonruter could you review and let me know what you think?

### CODE REVIEW

File: dependency-minification.php [hunk 1]
```diff
@@ -505,7 +505,7 @@ static function get_cache_option_name( $src_hash ) {
 	static function get_dependency_minified_url( array $deps, $type ) {
 		$src_hash = self::hash_array( wp_list_pluck( $deps, 'src' ) );
 		$ver_hash = self::hash_array( wp_list_pluck( $deps, 'ver' ) );
-		$src = trailingslashit( get_option( 'home' ) . DIRECTORY_SEPARATOR . self::$options['endpoint'] );
+		$src = esc_url( trailingslashit( site_url( self::$options['endpoint'] ) ) );
```
Comment (westonruter) [inline dependency-minification.php]: @topher1kenobe I don't believe this should be `esc_url` this here. It will get escaped later when output.

**Comment by westonruter** [comment]
Also fixes #50.

**Comment by westonruter** [comment]
See also #54.

File: .travis.yml [hunk 2]
```diff
@@ -1 +0,0 @@
-bin/.travis.yml
```
Comment (westonruter) [inline .travis.yml:1]: Why the deletion of this symlink?

**Comment by westonruter** [comment]
@topher1kenobe in think you meant to merge this on your fork instead.

**Comment by topher1kenobe** [comment]
Yeah, I'm working on fixing it.

**Comment by westonruter** [comment]
Done. I reset the branch back to c579d699e9ba9facdeee721675e51930c0c6f839.

**Comment by westonruter** [comment]
``` bash
git reset --hard c579d699e9ba9facdeee721675e51930c0c6f839
git push -f
```

**Comment by topher1kenobe** [comment]
I did as well.

**Pull request merged**